### DISCUSSION TITLE
How to access HasManyRepeater data from mutateFormDataBeforeSave()?

### DISCUSSION THREAD

**Discussion by otavio-araujo** [filamentphp/filament#2838] [Help] [answered]
How can i access HasManyRepeater's data from mutateFormDataBeforeSave()?

`protected function mutateFormDataBeforeSave(array $data): array
{
dd($data);
return $data;
}`

![image](https://user-images.githubusercontent.com/30505243/174495329-235f2d93-350c-49bf-9dae-3eb7dbfcf2f5.png)

**Comment by danharrin** [comment, marked as answer]
#2741

**Reply by otavio-araujo** [reply]
Thanks a lot!
